src/lightly_train/_modules/teachers/dinov2/models/vision_transformer.py

```diff
@@ -30,6 +32,19 @@
 
 logger = logging.getLogger(__name__)
 
+XFORMERS_INSTALLED = RequirementCache("xformers>=0.0.18") is not None
```

Comment:
```suggestion
XFORMERS_INSTALLED = RequirementCache("xformers>=0.0.18")
```
A `RequirementCache` object should never be `None`. It behaves like a boolean.

Reply:
yep, god catch. Forgot to remove it.